### ISSUE TITLE
Unconventional date formatting in Norwegian Bokmål

### ISSUE THREAD

**Issue by datafoo** [amcharts/amcharts5#310]
### Steps to reproduce

1. Configure a chart with a date axis and tooltip on this date axis, set the locale `root.locale = am5locales_nb_NO;` (see https://codepen.io/jackfoo/pen/qBVyrgG and zoom in to show different styled of formatting)
2. Hover over the series and notice the date format of both the axis labels and the axis tooltips 

### Actual result
The date formats do not use the conventions used in Norwegian Bokmål:

1. `Feb. 24`
2. `Feb. 24, 2022`
3. `16:00 - feb. 24, 2022`

### Expected result:
The date formats should use the conventions used in Norwegian Bokmål:

1. `24. feb.`
2. `24. feb. 2022`
3. `16:00 - 24. feb. 2022` or even better `24. feb. 2022, 16:00`

### More info

From my understanding (I am not Norwegian), in Norwegian Bokmål, the day always comes before the month. Moreover, a dot is placed right after the day number.

Example:
```js
const date = new Date(Date.UTC(2022, 1, 24, 11, 58, 0));
console.log(new Intl.DateTimeFormat('nb-NO', { dateStyle: 'short' }).format(date));
console.log(new Intl.DateTimeFormat('nb-NO', { dateStyle: 'medium', timeStyle: 'short' }).format(date));
```
On my browser, it displays:
```js
24.02.2022
24. feb. 2022, 12:58
```

**Comment by datafoo** [comment]
It seems amCharts 4 is also concerned by this issue.

**Comment by martynasma** [comment]
The Norwegian language file, including date formats, was prepared by a Norwegian person.

If you think it is not right, would you be able to point us to some official document outlining date/time formats used in Norway?

In the interim, you can configure your own date formats as needed via global DateFormatter as well as DateAxis formats:
https://www.amcharts.com/docs/v5/concepts/formatters/formatting-dates/
https://www.amcharts.com/docs/v5/charts/xy-chart/axes/date-axis/#date-formats

**Comment by datafoo** [comment]
> The Norwegian language file, including date formats, was prepared by a Norwegian person.
> 
> If you think it is not right, would you be able to point us to some official document outlining date/time formats used in Norway?

I do not know what is the official body who decides the rules but here is some documentation.

Information in English:

- https://www.babbel.com/en/magazine/how-to-write-the-date-in-norwegian
- https://leap.hcldoc.com/help/topic/SSS28S_8.2.1/XFDL_Specification/i_xfdl_r_formats_nb_NO.html

Information in Norwegian:

- https://www.sprakradet.no/sprakhjelp/Skriveregler/Dato/#dato
- https://norsksidene.no/web/PageND.aspx?id=99321
- https://no.wikipedia.org/wiki/Dato

**Comment by martynasma** [comment]
Fixed in [5.1.7](https://github.com/amcharts/amcharts5/releases/tag/5.1.7).

## [5.1.7] - 2022-02-28

### Fixed
- Grouping to periods where `timeUnit` was `"day"` and `count > 1` was not working properly.
- When tooltips of multiple series were pointing at the same point, series order was not being respected (FF only).
- If a scrollbar was added before axes, scrolling series could result series to be out of sync with bullets.
- Tweaked `nb_NO` locale to better correspond to Norwegian Bokmål date formats.

[Full change log](https://github.com/amcharts/amcharts5/blob/master/packages/shared/CHANGELOG.md).

[Download options](https://www.amcharts.com/download/).

Make sure you clear your browser cache after upgrading. And feel free to contact us again if you are still experiencing this issue.
